### ISSUE TITLE
Please include AQC113C

### ISSUE THREAD

**Issue by icekingcy** [Aquantia/AQtion-esxi#23]
`0000:69:00.0 Network controller Ethernet controller: Aquantia Corp. Device 14c0
         Class 0200: 1d6a:14c0
`

**Comment by dimbez** [comment]
Hi @icekingcy,
Sorry for delayed response. 
Support of AQC113 is under discussion. There is no estimate when/if it will be supported.

**Comment by minduch** [comment]
Yes, please add AQC113C that is the controller/driver needed for QNAP's QXG-10G2TB dual 10GBE NIC.

**Comment by AnonBit** [comment]
> Hi @icekingcy, Sorry for delayed response. Support of AQC113 is under discussion. There is no estimate when/if it will be supported.

Any news, will be supported ?

**Comment by dimbez** [comment]
Hi @AnonBit unfortunately still no updates on it.

**Comment by Breugeltje** [comment]
Since the ASUS ProArt X570-Creator mainbord has this network controller onboard and that mainbord is awesome for ESXI and even better when this network controller would be working, i would also be verry happy if there would be support.
by the way Marvell states on there site it's supported in ESXI but creating a support ticket there is kinda made impossible.

**Comment by NielsPiersma** [comment]
I would also opt for adding the AQC113C driver. Now running awsome Z690 AORUS XTREME (rev. 1.0) https://www.gigabyte.com/Motherboard/Z690-AORUS-XTREME-rev-10#kf which include both a 2.5GB Intel nic (commit supported driver) and a 10Gb AQC113C. Running two NICs would be superb. Especially at 10Gb !!!
Cheers

**Comment by SupremeLiam** [comment]
> Hi @AnonBit unfortunately still no updates on it.

Any news, will it be supported on ESXi?

**Comment by dimbez** [comment]
Hi @SupremeLiam 
support of aqc113 is not going to be introduced.

**Comment by SupremeLiam** [comment]
> 

So does it mean that all NICs using AQC113 will have no driver support for ESXi or Linux?

**Comment by dimbez** [comment]
For ESXi current status - no plan to introduce support.
For linux - it's supported, here we are talking about ESXi only.

**Comment by Breugeltje** [comment]
I still don't understand why Marvell advertises this chip to be supported in ESXi.
If it had not i would have bought a other mainboard with more pci-e ports so i could put in a supported 10G NIC.

**Comment by dimbez** [comment]
@Breugeltje, there are 2 generations of NICs - aqc107 based it's supported and aqc113 based it's not supported.

**Comment by Breugeltje** [comment]
@dimbez to be clear i don't know if you work for Marvell or not, but in the official datasheet for the AQC113 series ESXi is in the supported OS line. that is why i don't understand why Marvell does not do there part in actually support the OS with a driver
[marvell-fastLinq-edge-aqc113-aqc113c-aqc113cs-aqc114cs-aqc115c-aqc116c-product-brief.pdf](https://github.com/Aquantia/AQtion-esxi/files/9235000/marvell-fastLinq-edge-aqc113-aqc113c-aqc113cs-aqc114cs-aqc115c-aqc116c-product-brief.pdf)
.

**Comment by goldserve** [comment]
Please please reconsider and add support for the AQC113 in ESXI 6.7?

**Comment by ZSG-Studios** [comment]
Please please reconsider and add support for the AQC113

**Comment by RoboDoc** [comment]
Yes. Please, please reconsider and add support for the AQC113x in ESXi8.

**Comment by waazdakka** [comment]
Hello! :) Any hope of seeing this compatibility arrive one day? Since all the 10Gb card from Qnap are using this chipset, I guess there is some people with the same issue as us...
Who do we need to speak with to add it on the compatibility list?

**Comment by showxu** [comment]
Please reconsider and add support for the AQC113x in ESXi8.

**Comment by mamhuijb** [comment]
its a old thread but make the Marvell AQC113CS work in esxi 8 from you side its something than can be done fast

**Comment by hhjsasa** [comment]
Please please reconsider and add support for the AQC113 in ESXI 6.5

**Comment by waazdakka** [comment]
Add this point, better consider another hypervisor 🥲

**Comment by dimbez** [comment]
Hi, this product is in support state only, no further development is planned as I know.
